recipes/default.rb

```diff
@@ -43,7 +38,7 @@
   only_if { node['platform_family'] == 'debian' && node['platform_version'].to_i >= 9 }
 end
 
-apt_repository 'docker-main' do
+apt_repository 'docker-stable' do
```

Comment:
This should probably be converted to using the ``chef-apt-docker`` cookbook instead.